test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/AssociatedValidatorProviderTest.cs

```diff
@@ -76,6 +76,73 @@ public void GetValidatorsForPropertyWithMixedAttributes()
             Assert.True(callbackAttributes.Any(a => a is RequiredAttribute));
         }
 
+        [Fact]
+        public void GetModelMetadataTypeAttributeValidators_DifferentAttributesOnProperty()
```

Comment:
add expectations to test names in this class e.g. `GetValidators_WithDifferentAttributesOnProperty_ReturnsAll()`